### ISSUE TITLE
Activating the 'registrants tab, displays a table with the details of the registrants. This looks like a table, however screen reader doesn't announce the data in a tabular format.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3235]
**Describe the bug**

Activating the 'registrants tab, displays a table with the details of the registrants. This looks like a table, however screen reader doesn't announce the data in a tabular format.

**To Reproduce**

"1. Go to the sample challenge page
2. Ensure that the screen reader is switched on. 
3. Navigate to the 'registrants' control using the tab key or the arrow key. 
4. Press enter on the 'registrants' control. 
5. Press the down arrow to access the registrants information. 
6. The information is displayed in a tabular form, however screen reader doesn't recognise the tabular format."

**Expected behavior**

Native tabular structure should be used so as the screen reader reads the data in a tabular format.

**Actual behavior**

Screen reader reads the registrants information in a textual format and not in a tabular format.

**Additional context**

Type: Registrants

Page:https://www.topcoder.com/challenges/30095347

Status: Not Fixed

**Comment by drasticdpk** [comment]
Verified that issue is no longer exist with screen reader JAWS.
